src/main/Catalog.js

```diff
@@ -1,27 +1,29 @@
 import React, { useState } from "react";
 
+const url = "https://api.punkapi.com/v2/beers";
```

Comment:
Let's imagine you have multiple URLs to use for fetching, how would you differentiate with such naming? 🙂